packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -1738,7 +1738,7 @@ export abstract class Chart extends Observable implements ModuleInstance, ChartS
 
         for (let index = 0; index < options.length; index++) {
             const axisOptions = options[index];
-            const axis = axisRegistry.create(axisOptions.type, moduleContext);
+            const axis = ModuleRegistry.getAxisModule(axisOptions.type)!.create(moduleContext) as any;
```

Comment:
Are the `as any` casts a temporary measure?

Reply:
Yes, it's temporary, though we should probably find a better way to handle this.